### PR TITLE
Released version 3.0.12

### PR DESCRIPTION
**Pull request by DaleCantwell**
* Adding a new method to the public API `displayMessageComposerWithInitialMessage`
* Updated Intercom for Android to [3.0.10](https://github.com/intercom/intercom-android/releases/tag/3.0.10).
* Updated Intercom for iOS to [3.0.13](https://github.com/intercom/intercom-ios/releases/tag/3.0.13).
* Localization updates.

### CODE REVIEW

File: src/ios/IntercomBridge.m [hunk 1]
```diff
@@ -9,7 +9,7 @@ + (void)setCordovaVersion:(NSString *)v;
 @implementation IntercomBridge : CDVPlugin
 
 - (void)pluginInitialize {
-    [Intercom setCordovaVersion:@"3.0.11"];
+    [Intercom setCordovaVersion:@"3.0.13"];
```
Comment (bboyle18) [inline src/ios/IntercomBridge.m]: Think this should be 3.0.12?

Comment (DaleCantwell) [inline src/ios/IntercomBridge.m]: Updating this now.

Comment (jawgardner) [inline src/ios/IntercomBridge.m]: npm is failing on install now:

`npm ERR! fetch failed https://registry.npmjs.org/cordova-plugin-intercom/-/cordova-plugin-intercom-3.0.12.tgz`

Comment (jtreanor) [inline src/ios/IntercomBridge.m]: @jawgardner How are you installing the plugin?

**Pull request merged**